salt/states/module.py

```diff
@@ -319,13 +319,10 @@ def _call_function(name, returner=None, **kwargs):
     arg_type, na_type, kw_type = [], {}, False
```

Comment:
`kw_type` is seems unneeded anymore.

Reply:
Yeah, when looking at the [whole file](https://github.com/cachedout/salt/blob/cd6f0dd08c9afe633e2781fdc169cc73802b87f7/salt/states/module.py) it seems like he only code that references `kw_type` is gone.